demos/quiz_bot/README.md

```diff
@@ -0,0 +1,73 @@
+# HammettQuizBot
+
+This demo showcases the `SingleChoiceWidget` and `MultiChoiceWidget` screens provided by Hammett. It allows you to create screens with single or multiple choices. Check out the documentation for more details.
```

Comment:
This demo showcases the `SingleChoiceWidget` and `MultiChoiceWidget` widgets to tackle various tasks. It demonstrates how they can be used to create a quiz with options for selecting one or multiple correct answers. Additionally, the bot illustrates how to use the `SingleChoiceWidget` and `.po` files to create multilingual interfaces.